collector/lib/FileDownloader.cpp

```diff
@@ -136,6 +137,9 @@ FileDownloader::~FileDownloader() {
 }
 
 bool FileDownloader::SetURL(const char* const url) {
+  url_path_ = std::string(url);
+  url_path_ = url_path_.substr(url_path_.find_last_of("/") + 1);
```

Comment:
We already log the full URL so this is to make subsequent log messages a bit shorter and clearer, since this will contain the file we're trying to download. I think the full URL message is actually a debug message so I think I'll make it INFO instead